aws-rds-dbclusterparametergroup/src/main/java/software/amazon/rds/dbclusterparametergroup/Translator.java

```diff
@@ -164,19 +83,22 @@ else if (parameter.applyType().equalsIgnoreCase(DYNAMIC_TYPE)) {
                 .collect(Collectors.toSet());
     }
 
-    static Set<software.amazon.rds.dbclusterparametergroup.Tag> mapToTags(final Map<String, String> tags) {
-        return Optional.of(tags.entrySet()).orElse(Collections.emptySet())
-                .stream()
-                .map(entry -> software.amazon.rds.dbclusterparametergroup.Tag.builder().key(entry.getKey()).value(entry.getValue()).build())
-                .collect(Collectors.toSet());
-    }
-
-    static Set<software.amazon.rds.dbclusterparametergroup.Tag> translateTagsFromSdk(final Collection<Tag> tags) {
+    static List<software.amazon.rds.dbclusterparametergroup.Tag> translateTagsFromSdk(final Collection<Tag> tags) {
```

Comment:
My very opinionated pov: i personally prefer to use full class names for the ones we import from awssdk model. `software.amazon.rds.dbclusterparametergroup.Tag` is somewhat a local class, hence, IMO, it is more natural to flip the import fqdn the other way around:

```java
static List<Tag> translateTagsFromSdk(final Collection<software.amazon.awssdk.services.rds.model.Tag> tags) {
```